docs/AboutUs.md

```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Wang Tao
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/wangtao.png" width="200px">
```

Comment:
Hi @wangtao0717 I believe you should change the URL to your github_username_in_lower_case as specified on the website. Otherwise this LGTM!